src/test/java/tool/designpatterns/verifiers/multiclassverifiers/AdapterVerifierTest.java

```diff
@@ -0,0 +1,100 @@
+package tool.designpatterns.verifiers.multiclassverifiers;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.gradle.internal.impldep.org.junit.Assert.assertFalse;
+import static org.gradle.internal.impldep.org.junit.Assert.assertTrue;
+
+import com.github.javaparser.ast.NodeList;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.type.ClassOrInterfaceType;
+
+import org.junit.jupiter.api.Test;
+import tool.designpatterns.Pattern;
+import utilities.TestHelper;
+
+public class AdapterVerifierTest {
+
+    @Test
+    public void testVerifyImplementsAdapter() throws IOException {
+        ClassOrInterfaceDeclaration adapter = TestHelper.getMockCompUnit(
+            "adapter" + "/implementsAdapter", "MicroUsbToLightningAdapter").findAll(
+            ClassOrInterfaceDeclaration.class).get(0);
```

Comment:
There is a helpermethod getMockClassOrI which does what you do.

Reply:
Fixed